addons/web/static/src/views/kanban/kanban_column_quick_create.xml

```diff
@@ -15,9 +15,10 @@
                             class="form-control o_input bg-transparent fs-4"
                             t-attf-placeholder="{{ relatedFieldName }}..."
                             t-ref="autofocus"
-                            t-model.trim="state.columnTitle"
+                            t-model="state.columnTitle"
```

Comment:
you can remove the t-model and the state, I think. You can directly read in the input in validate